Unclear, but dev-env may be failing on this

```
629.5 Collecting automat==25.4.16
629.5   Downloading automat-25.4.16-py3-none-any.whl (42 kB)
629.6 ERROR: Could not find a version that satisfies the requirement awx-plugins-interfaces (unavailable) (from versions: 0.0.1a1, 0.0.1a2, 0.0.1a3, 0.0.1a4, 0.0.1a5)
629.6 ERROR: No matching distribution found for awx-plugins-interfaces (unavailable)
629.6 WARNING: You are using pip version 21.2.4; however, version 25.3 is available.
629.6 You should consider upgrading via the '/var/lib/awx/venv/awx/bin/python3.11 -m pip install --upgrade pip' command.
629.7 make: *** [Makefile:183: requirements_awx] Error 1
```

doesn't look related

--- a/awx/main/management/commands/run_dispatcher.py
+++ b/awx/main/management/commands/run_dispatcher.py
@@ -2,7 +2,6 @@
 # All Rights Reserved.
 import logging
 import yaml
-import os
 
 import redis
 

--- a/requirements/requirements.in
+++ b/requirements/requirements.in
@@ -71,7 +71,7 @@ setuptools_scm[toml]
 setuptools-rust>=0.11.4  # cryptography build dep
 pkgconfig>=1.5.1  # xmlsec build dep - needed for offline build
 django-flags>=5.0.13
-dispatcherd  # tasking system, previously part of AWX code base
+dispatcherd[pg_notify]  # tasking system, previously part of AWX code base
 protobuf>=4.25.8 # CVE-2025-4565
 idna>=3.10 # CVE-2024-3651
 # Temporarily added to use ansible-runner from git branch, to be removed

--- a/requirements/requirements.txt
+++ b/requirements/requirements.txt
@@ -120,7 +120,7 @@ cython==3.1.3
     # via -r /awx_devel/requirements/requirements.in
 daphne==4.2.1
     # via -r /awx_devel/requirements/requirements.in
-dispatcherd==2025.5.21
+dispatcherd[pg_notify]==2025.12.10
     # via -r /awx_devel/requirements/requirements.in
 distro==1.9.0
     # via -r /awx_devel/requirements/requirements.in
@@ -138,7 +138,7 @@ django==5.2.8
     #   django-solo
     #   djangorestframework
     #   drf-spectacular
-# django-ansible-base[feature-flags,jwt-consumer,rbac,resource-registry,rest-filters] @ git+https://github.com/ansible/django-ansible-base@devel  # git requirements installed separately
+# django-ansible-base @ git+https://github.com/ansible/django-ansible-base@devel  # git requirements installed separately
     # via -r /awx_devel/requirements/requirements_git.txt
 django-cors-headers==4.9.0
     # via -r /awx_devel/requirements/requirements.in

